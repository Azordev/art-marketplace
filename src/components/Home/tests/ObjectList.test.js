import React from "react";
import { MemoryRouter } from "react-router";
import renderer from "react-test-renderer";
import ObjectList from "../ObjectList";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <ObjectList title="test" seeMore="#" items={[]} />
      </MemoryRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
