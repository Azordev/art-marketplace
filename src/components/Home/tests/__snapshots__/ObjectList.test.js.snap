// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="w-full font-bold flex justify-start items-end px-10 lg:px-20 xl:px-10 pt-2 pb-6 flex-wrap"
  >
    <p
      className="text-3xl sm:text-4xl pr-5 py-2 xl:mr-4 sm:py-0"
    >
      test
    </p>
    <a
      className="text-2xl tracking-tighter font-normal text-gray-600"
      href="/"
      onClick={[Function]}
    >
      Ver más
    </a>
  </div>
  <div
    className="w-full flex flex-col flex-wrap xl:flex-nowrap md:flex-row md:justify-evenly lg:justify-evenly xl:justify-between xl:space-x-3 2xl:space-x-0 px-5 md:px-0 lg:px-20 xl:px-12 py-6 xl:pb-20"
  >
    <span
      className="text-xl text-gray-500"
    >
      Actualmente no hay productos para mostrar en esta sección
    </span>
  </div>
</div>
`;
